views/artist-search-results.hbs

```diff
@@ -0,0 +1,11 @@
+
+  {{#each findArtists}}
+  <div>
+    <h1>{{this.name}}</h1>
+    <button><a href ="/albums/{{this.id}}" class="btn btn-primary"></a> View Albums</a>  </button> 
```

Comment:
should remove the </a> before the label for the button to work properly
`    <button><a href ="/albums/{{this.id}}" class="btn btn-primary"> View Albums</a>  </button> `